app/controllers/admin/events_controller.rb

```diff
@@ -27,6 +27,7 @@ def show
     @attending_students = InvitationPresenter.decorate_collection(@original_event.attending_students)
     @attending_coaches = InvitationPresenter.decorate_collection(@original_event.attending_coaches)
     @host_address = AddressPresenter.new(@event.venue.address)
+    @chapters = @event.chapters.asc(:name)
```

Comment:
This breaks the app. I think chapters need to be retrieved from `@original_event`